sympy/utilities/autowrap.py

```diff
@@ -527,8 +609,8 @@ def _validate_backend_language(backend, language):
 
 @cacheit
 @doctest_depends_on(exe=('f2py', 'gfortran'), modules=('numpy',))
-def autowrap(expr, language=None, backend='f2py', tempdir=None, args=None,
-             flags=None, verbose=False, helpers=None, code_gen=None, **kwargs):
+def autowrap(expr_or_routines, language=None, backend='f2py', tempdir=None, args=None,
+             flags=None, verbose=False, helpers=None, code_gen=None, user_local_vars=None, settings={}, **kwargs):
```

Comment:
we should not use [mutable default values](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) keyword arguments (they are essentially global variables in python, unfortunately). You can set de default to `None` and then upon first access use the idiom `settings or {}`.